docs/sources/write/style-guide/inclusive-writing/index.md

```diff
@@ -1,79 +1,88 @@
 
 ## Writing about people
 
 When writing about people, be compassionate, inclusive, and respectful.
 
-First and foremost, no matter the context, we always make sure to view people as people. When context and word count permits, we refer to them as people or people who maintain a particular area or discipline.
-
+No matter the context, we always make sure to view people as people.
 With that in mind, there are important distinctions between the ways that different people interact with our product and, thus, how we refer to those people.
 
 - Users are every person who uses our products, whether or not they use a commercial version of one of our products. Customers are people and companies who pay to use our products.
 - Visitors are people who visit our website for any reason, whether it's as a potential user, customer, or employee. We appreciate all interest in our hard work.
-- Our customers (and users) often have their own customers (and users). Generally speaking, we refer to these customers of our customers as users, because it does an okay job of encapsulating all the ways our various customers might interact with the people using their products.
+- Our customers (and users) often have their own customers (and users).
+  Generally speaking, we refer to these customers of our customers as users, because it does an OK job of encapsulating all the ways different customers might interact with the people using their products.
 - Employees are people who work at Grafana Labs.
 
 ### Best practices
 
 Consider the following best practices to ensure inclusivity in your writing.
 
-- Group or audience: Don’t refer to a group or audience as _it_. Instead, use _they_.
-- Age: Don’t refer to someone’s age unless it’s relevant. Don’t use age-related words to describe people. Avoid: _young_, _old_, or _elderly_.
+- Group or audience: Don’t refer to a group or audience as _it_.
+  Instead, use _they_.
+- Age: Don’t refer to someone’s age unless it’s relevant.
+  Don’t use age-related words to describe people.
+  Avoid: _young_, _old_, or _elderly_.
 - Hearing: Do use the adjectives _deaf_, _partially deaf_, or _hard of hearing_ when describing a person with hearing loss.
 - Vision: Do use the adjective _blind_ or _low vision_ when describing a person with limited vision.
 
 ## Write for an international audience
 
-Grafana Labs uses US English as our standard (for spelling, grammar, punctuation, and more).
+Grafana Labs uses U.S. English as our standard for spelling, grammar, punctuation, and more.
 We write for an international audience with the following guidelines in mind:
 
 ### Avoid acronyms
 
 If you use an acronym, first write out each word in the acronym on first use, and then continue using just the acronym.
 
-- **Example**: Grafana Cloud is software as a service, also known as SaaS. Using SaaS typically requires setting up an account.
+- **Example**: Grafana Cloud is software as a service (SaaS).
```

Comment:
```suggestion
- **Example**: Grafana Cloud is software-as-a-service (SaaS).
```

Reply:
Wikipedia doesn't hyphenate this word: https://en.wikipedia.org/wiki/Software_as_a_service

Reply:
ah, ok - I think I'm just used to seeing the hyphenated form from many years ago (perhaps? It was a thing to hyphenate at some point?). All good, you can disregard my suggestion. Thank you!